multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BuildApplicationDeployModelStep.java

```diff
@@ -87,6 +87,7 @@ private Staging modifyHealthCheckType(Staging staging) {
 
     private List<String> getHealthCheckValues() {
         return Arrays.stream(HealthCheckType.values())
+                     .filter(healthCheckType -> healthCheckType != HealthCheckType.NONE)
```

Comment:
What does this have to do with the tokens? :D

Reply:
Nothing at all and respectively is in another commit